tests/src/PublicClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
Can you move this class/interface into the test file `namespaceUsagesExcludeAttribute.php` as well?

Reply:
Moved over... it breaks composers autoloading when using `class_exists` or `interface_exists` - but those checks aren't required since we're using PHPStan's reflector which is much more robust - I've still added a try / catch just in case the Reflector can't find the class which mimics the original behaviour.